webapp/sources/rudder/rudder-web/src/main/gulpfile.js

```diff
@@ -9,15 +9,24 @@ const elm_p = require('gulp-elm');
 const merge = require('merge-stream');
 const del = require('del');
 const through = require('through2');
+const sass = require('gulp-sass')(require('sass'));
+const sourcemaps = require('gulp-sourcemaps');
 
 const paths = {
     'css': {
-        'src': 'style/**/*',
+        'src': [
+          'style/libs/**/*',
+          'style/login.css'
+        ],
         'dest': 'webapp/style/',
     },
+    'scss': {
```

Comment:
you could make a `login_scss` with its own src and dest, and use the same `scss` function for both.

Reply:
Thanks, I did that and created 2 streams in the scss function, one for the "Rudder" scss files and the other for the login scss file. Then I merged these two streams